src/pages/community-partners.astro

```diff
@@ -0,0 +1,94 @@
+---
+import Layout from '@layouts/Default.astro'
+
+const partners = [
+  {
+    'name': 'The Algorithms',
+    'image': 'https://github.com/TheAlgorithms.png',
+    'link': 'https://the-algorithms.com',
+    'description': 'Open Source resource for learning Data Structures & Algorithms and their implementation in any Programming Language'
+  },
+  {
+    'name': 'GitHub',
+    'image': 'https://github.com/GitHub.png',
+    'link': 'https://github.com',
+    'description': 'GitHub is one of the world’s largest community of developers. It’s an intricate platform that fosters collaboration and communication between developers.'
+  },
+  {
+    'name': 'Web3Community',
+    'image': 'https://github.com/Web3Community.png',
+    'link': 'https://web3community.github.io',
+    'description': 'An Open Source Community that focuses on decentralized applications, web 3, and blockchain technologies 🚀'
```

Comment:
```suggestion
    'description': 'An Open Source Community that focuses on decentralized applications, Web 3.0, and Blockchain technologies 🚀'
```

Reply:
I think it should be lowercase (#Grammarly)